framework/src/modules/interoperability/index.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+// Mainchain
+export { MainchainInteroperabilityModule } from './mainchain/module';
+export { MainchainInteroperabilityAPI } from './mainchain/api';
+export * from './mainchain/commands';
```

Comment:
some command name over-wraps with the sidechain ones?

Reply:
There was one conflicting name that I resolved already here [here](https://github.com/LiskHQ/lisk-sdk/pull/7292/commits/b31d1a498d2b3256af7f31e0f7b533b7f041e3dc#diff-5523e0a9c42ea84f5a360733d8fc2e03471bac2a986790fb9bbfcec8c2f19757R43). Otherwise all the commands starts with `mainchain` or `sidechain` prefix or except for `stateRecovery` and `stateRecoveryInit` which is only in Mainchain